src/index.js

```diff
@@ -12,11 +12,6 @@ const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
 export const defaultExtensions = defaultBabelExtensions;
 
 export function mapModule(sourcePath, currentFile, pluginOpts, cwd) {
-  // Do not map source starting with a dot
```

Comment:
`getRealPath` already does that check, so it didn't make sense to check this special case here, especially since the empty paths are also being ignored, but they are not being checked here.